src/patternfly/components/Nav/examples/nav-example.hbs

```diff
@@ -10,6 +10,7 @@
         Link 2
       {{/nav-link}}
     {{/nav-item}}
+    {{#> nav-item nav-item--modifier="pf-m-separator" nav-item--attribute='role="separator"'}}{{/nav-item}}
```

Comment:
In the options menu we also have a separator with the class - pf-c-options-menu__separator. I wonder if we should update that component and potentially other components to use the modifier like here. And would we add a utility class for this?

Reply:
Ah good catch! I just used what was already in the nav component for this. Personally I think it's better as it's own element, and not as a modifier. If we were to add any sort of styles to a regular nav-item (or options menu item, etc) like padding, borders, background color, hover/focus treatment, etc, I can't imagine we would ever want those styles to apply to the separator. So in the CSS, we would need to use `:not(.pf-m-separator)` or whatever to target non-separator items. To me a regular item and separator item are separate elements. WDYT?

And can you give me an example of how we would benefit from adding this as a utility class? The approach so far has been to not create a utility class unless it's obviously useful, and instead wait for folks to request one.

Reply:
I agree with what you said above I dont think it should be a modifier. Would we make the separator a component? or just an element in the nav?

To me a separator has the same properties as a border. For example in the data table and data list examples I had to add extra borders to the component in order to meet the design specs, however maybe a separator would have been better there? I can see a separator utility being added to a lot of cases to separate content or add an extra border between two elements. But in that case, what if we made it its own component?